.config/s3.config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if (getenv('OBJECTSTORE_S3_HOST')) {
+  $CONFIG = array (
+    'objectstore' => array(
+      'class' => '\\OC\\Files\\ObjectStore\\S3',
+      'arguments' => array(
+        'bucket' => getenv('OBJECTSTORE_S3_BUCKET'),
+        'autocreate' => getenv('OBJECTSTORE_S3_AUTOCREATE') ?: true,
+        'key'    => getenv('OBJECTSTORE_S3_KEY'),
+        'secret' => getenv('OBJECTSTORE_S3_SECRET'),
+        'hostname' => getenv('OBJECTSTORE_S3_HOST'),
+        'port' => getenv('OBJECTSTORE_S3_PORT'),
+        'use_ssl' => getenv('OBJECTSTORE_S3_SSL') ?: true,
+        'region' => getenv('OBJECTSTORE_S3_REGION'),
```

Comment:
I'm doing some testing of this. If we do this here like this it ends up in the final config.php like this:

```
      'hostname' => false,
      'port' => false,
      'region' => false,
```
The app seems to handle it okay (probably because it matches up okay in a weak null check).